testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java

```diff
@@ -773,5 +773,116 @@ public void verifyFT85() throws IOException,
         assertTrue(expectedJsonResponse.equals(EntityUtils.toString(response.getEntity()))  );
 
     }
-    
+
+    @Test
+    public void verifyFT117() throws IOException, InterruptedException {
+        /**
+         * To check NMS is able to make available a single message of current
+         * week in inbox with single message per week configuration . when a)
+         * user's MSISDN is subscribed for 48Weeks Pack. b)user's MSISDN is
+         * deactivated for an old subscription of 72Weeks Pack.
+         **/
+        // setup data to remove 2 messages per week configuration for 72 week pack
+        testingService.clearDatabase();
```

Comment:
setupData() like 114 takes care to delete the database and create rh & sh.
No need to do the same here.